source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/filters/filters.scss

```diff
@@ -7,14 +7,6 @@
   margin-bottom: 40px;
```

Comment:
should we use m-t-(breakpoint here?)

Reply:
probably, yeah. m-t-size-maybe-breakpoint.